commit b1e97f5b26a1c2c0980191f8c68b4e6c0bcff4f6
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2019-09-13T16:29:30+02:00 (3 weeks ago)
    
    TRITON-1894 svc:/smartdc/agent/firewaller-config-migration:default goes into maint

diff --git a/config-migration.js b/config-migration.js
index c1f5644..d9aa4db 100644
--- a/config-migration.js
+++ b/config-migration.js
@@ -77,6 +77,10 @@ vasync.pipeline({
                 func: function testIPFFiles(vm, nextVm) {
                     var ipfv4 = util.format(IPF_CONF,
                         ctx.vmsByVmUUID[vm.uuid].zonepath);
+                    if (!fs.existsSycn(ipfv4)) {
+                        nextVm();
+                        return;
+                    }
                     var ipfv4Data = fs.readFileSync(ipfv4, 'utf8');
                     var ipfv4Res = IPF_VER_RE.exec(ipfv4Data);
                     // If we cannot find a version written on the rules file,
